community-details.rst

```diff
@@ -11,18 +11,18 @@ Storj has an active and growing community that is over 150,000 members. It's a m
 Rocket Chat Channel
 ~~~~~~~~~~~~~~~~~~~
 
-`Rocket Chat`_ is our community chat where you can post specific topics related to development, farming, partnerships, storj share, payments, and etc. There are specific channels where these topics are addressed.  We also offer language-specific support that spans over 8 languages.  If you're bilingual and like to moderate a language channel that isn't supported, please reach out to bounty@storj.io.  Not sure who to reach out to in Rocket Chat? Below is a list of our admins. 
+`Rocket Chat`_ is our community chat where you can post specific topics related to development, setting up farming nodes, partnerships, Storj Share, payouts, billing etc. There are specific channels where these topics are addressed.  We also offer language-specific support that spans over 8 languages.  If you're bilingual and like to moderate a language channel that isn't supported, please reach out to bounty@storj.io.  Not sure who to reach out to in Rocket Chat? Below is a list of our admins. 
 
   .. _Rocket Chat: https://storj.io/community.html
 
 @knowledge
 	* some text
 @heunland 
-	* some text
+	* Helene holds an A.A.S. in Computer Science, and a B.S. and M.Sci. in Hydrology and Water Resources. She is well versed in the latest developments in the crypto-currency space. She coordinates relations between the Storj team and its Community, directs the team of Community moderators and attends all types of customer support questions on Storj's social media and other support channels in three languages. You can ask her anything about Storj, crypto trading and safe management of your STORJ tokens.
 @littleskunk 
 	* some text
 @meije 
-	* Meije is Dutch and resides in Rotterdam. He speaks five languages and is active on most of our communication platforms. He is a geology master student at Utrecht University with a strong interest in blockchain, mapping, renewable energy, UAV's, and VR. He loves gaining a deep understanding on the interaction between the user and the available applications to be able to better serve the user.
+	* some text
```

Comment:
Meije is Dutch and resides in Rotterdam. He speaks five languages and is active on most of our communication platforms. He is a geology master student at Utrecht University with a strong interest in blockchain, mapping, renewable energy, UAV's, and VR. He loves gaining a deep understanding on the interaction between the user and the available applications to be able to better serve the user.